src/renderers/common/Renderer.js

```diff
@@ -2031,7 +2040,18 @@ class Renderer {
 	 */
 	get currentColorSpace() {
 
-		return this._renderTarget !== null ? LinearSRGBColorSpace : this.outputColorSpace;
+		return this.isOutputTarget ? this.outputColorSpace : LinearSRGBColorSpace;
+
+	}
+
+	/**
+	 * Returns `true` if the rendering settings are set to screen output.
+	 *
+	 * @returns {boolean} True if the current render target is the same of output render target or `null`, otherwise false.
+	 */
+	get isOutputTarget() {
+
+		return this._renderTarget === this._outputRenderTarget;
```

Comment:
I have not tested this change with a Quest and `webgpu_xr_cubes` yet. But when no post processing is used, does the default tone mapping and color space conversion work now with `webgpu_xr_cubes`?

What seems to be missing is to set `renderer.xr.enabled` to `false` in `_renderOutput()` when rendering the full screen quad and then set it back to the current value. Meaning the bit from:

https://github.com/mrdoob/three.js/pull/30387/files#diff-430a31e72073f4d737affbd06512c897d38b640a2a35fbfbb9464c9512b17105R1397